docs/generators/python-experimental.md

```diff
@@ -12,6 +12,7 @@ sidebar_label: python-experimental
 |packageUrl|python package URL.| |null|
 |packageVersion|python package version.| |1.0.0|
 |projectName|python project name in setup.py (e.g. petstore-api).| |null|
+|setRecursionLimit|Set recursion limit. If not set, use the system default value.| |null|
```

Comment:
What do you think about naming it recursionLimit?
other parameters only have verb prefixes if they are booleans. If you want to keep it as is that's fine also.

Reply:
Thanks for the feedback. Let me revise it.

Reply:
Updated via https://github.com/OpenAPITools/openapi-generator/pull/7491/commits/d84cdc50619e7284eb7c9ddc4d28ac0e3b143f06